crunchy-bridge-for-analytics/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# disable cache manager to prevent auto-loading of files
+$SUPERUSERCONNCMD -c 'ALTER SYSTEM SET crunchy_query_engine.enable_cache_manager TO off;'
```

Comment:
Just for my curiosity: Is this comment somewhere documented? At least Google does not know about it: https://www.google.com/search?q=crunchy_query_engine+enable_cache_manager&client=firefox-b-d&sca_esv=9267af3241730e66&ei=586sZpDcH52K9u8Ph-KTqAw&ved=0ahUKEwjQ6_DApdaHAxUdhf0HHQfxBMUQ4dUDCBE&uact=5&oq=crunchy_query_engine+enable_cache_manager&gs_lp=Egxnd3Mtd2l6LXNlcnAiKWNydW5jaHlfcXVlcnlfZW5naW5lIGVuYWJsZV9jYWNoZV9tYW5hZ2VyMggQABiABBiiBEiXCFCFBljdBnABeACQAQCYAZkBoAGJAqoBAzAuMrgBA8gBAPgBAZgCAaACmgGYAwCIBgGSBwMwLjGgB-oF&sclient=gws-wiz-serp

Reply:
I think it is not documented in our docs. 

In practice, no user would ever really need this. On all the instance types that we provide on Crunchy Bridge for Analytics, the local file caching is always enabled. That's probably why we haven't documented it (yet).